### PR TITLE
refactor: reorg elements project

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Reorg elements project
- deprecate original elements codes (from console & elements package)
- move account elements into elements project
- remove original `account-elements` package

Current project:
image

## Testing

No need.

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
~~- [ ] necessary TSDoc comments~~

### CODE REVIEW

File: packages/elements/xliff/ar-AR.xlf
Comment (charIeszhao) [inline packages/elements/xliff/ar-AR.xlf:1]: Seems these i18n phrases are just deleted?

File: packages/elements/src/account/types.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { type UserProfileResponse } from '@logto/schemas';
+
+export type UserProfile = Partial<UserProfileResponse>;
```
Comment (charIeszhao) [inline packages/elements/src/account/types.ts:3]: nit: Why partial? Maybe we can use pick instead?

File: packages/elements/src/account/api/index.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+export type GetAccessToken = () => Promise<string>;
+
+/**
+ * The API client for the Logto account elements
+ *
+ * Used to interact with Account-related backend APIs, including the Profile API.
+ */
+export class LogtoAccountApi {
+  private readonly ky: KyInstance;
```
Comment (charIeszhao) [inline packages/elements/src/account/api/index.ts:13]: nit: Can we get rid of ky from our elements package?

**Review by charIeszhao — APPROVED** [review]
Overall looks OK

File: packages/elements/xliff/ar-AR.xlf
Comment (xiaoyijun) [reply packages/elements/xliff/ar-AR.xlf:1]: Yes, we will have new i18n configurations.

File: packages/elements/src/account/types.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/elements/src/account/types.ts:3]: The backend returns partial object.

File: packages/elements/src/account/api/index.ts (hunk 2 above)
Comment (xiaoyijun) [reply packages/elements/src/account/api/index.ts:13]: It’s more convenient to implement the functionality using ky. We can evaluate later whether to remove it, but for now, let’s move on.

**Pull request merged**